.gitignore

```diff
@@ -2,8 +2,5 @@
 .DS_Store
 node_modules
 
-# Only apps should have lockfiles
 npm-shrinkwrap.json
-package-lock.json
-yarn.lock
-dist
```

Comment:
`dist` should still be listed to avoid the folder to be commit.
At the same time, you should remove it the `dist` folder you sent on this PR